.github/workflows/build.yml

```diff
@@ -30,6 +30,10 @@ jobs:
       run: |
         sbt '+kyoJVM/test'
 
+    - name: Build kyo-scheduler-finagle
```

Comment:
Finagle doesn't publish Scala 3 artifacts so the module is set only to Scala 2.13. I spent a few hours trying to make the SBT work but it always compiles the module even if on Scala 3. As a workaround, the new module isn't aggregated in `kyoJVM` and the workflows build the module individually.